bundle/ios/assets/jetpack/projects/plugins/jetpack/extensions/index.json

```diff
@@ -38,7 +38,7 @@
 		"videopress",
 		"wordads"
 	],
-	"beta": [ "amazon" ],
+	"beta": [ "amazon", "anchor-fm", "conversation", "dialogue" ],
```

Comment:
Just outta curiousity, we also are adding new things here?

Reply:
This file was auto-generated when I ran `npm run bundle:ios`, I'm not familiar with the release process yet but I think all files generated when making the bundle are committed, is that correct @cameronvoell?

I don't have enough context to know the implications of this change to be honest, my impression is that these are new beta extensions added recently in Jetpack but not necessarily will be added to our project.

Reply:
Fair enough, thanks!